quickwit/quickwit-cli/Cargo.toml

```diff
@@ -1,14 +1,13 @@
 [package]
 name = "quickwit-cli"
-version = "0.7.1"
-authors = ["Quickwit, Inc. <hello@quickwit.io>"]
-edition = "2021"
-license = "AGPL-3.0-or-later"                               # For a commercial, license, contact hello@quickwit.io
-description = "Quickwit is a cost-efficient search engine."
-repository = "https://github.com/quickwit-oss/quickwit"
-homepage = "https://quickwit.io/"
-documentation = "https://quickwit.io/docs/"
-default-run = "quickwit"
+
+version.workspace = true
+edition.workspace = true
+homepage.workspace = true
+documentation.workspace = true
+repository.workspace = true
+authors.workspace = true
+license.workspace = true
```

Comment:
maybe we want to keep `default-run = "quickwit"`?

Reply:
Good catch. I didn't notice that one.